ddp-workspace/projects/ddp-atcp/src/assets/i18n/en.json

```diff
@@ -0,0 +1,799 @@
+      "CREATED": "Not Started"
+    }
+  },
+  "SDK": {
+    "JoinUsButton": "Join Us",
+    "SubmitButton": "Submit",
+    "SaveAndSubmit": "Save & Submit",
+    "CloseButton": "Close",
+    "AddButton": "ADD",
+    "CancelButton": "Cancel",
+    "NextButton": "Next",
+    "PreviousButton": "Prev",
+    "SavingButton": "Saving",
+    "EditButton": "View/Edit",
+    "ReviewButton": "View",
+    "RegisterButton": "Register",
+    "SignAndConsent": "Sign & Consent",
+    "SignAndAssent": "Sign & Assent",
+    "DoNotConsent": "Do Not Consent",
+    "DoNotAssent": "Do Not Assent",
+    "StartOrResume": "Start/Resume",
+    "SaveAndSubmitEnrollment": "Save & Submit Enrollment",
+    "ValidateError": "Submission cannot proceed. Please review messages in form for details.",
+    "CommunicationError": "There was a problem saving your data. Please check the form and your answers before submitting",
+    "SaveError": "We're sorry but an error occurred",
+    "ExportStudy": {
+      "Export": "EXPORT"
+    },
+    "ListStudies": {
+      "StudyName": "Study Name",
+      "ParticipantCount": "Participant Count"
+    },
+    "ListWorkspaces": {
+      "WorkspaceName": "Workspace Name",
+      "WorkspaceNamespace": "Workspace Namespace"
+    },
+    "Login": {
+      "Login": "LOG IN",
+      "Logout": "LOG OUT"
+    },
+    "SignInOut": {
+      "SignIn": "Log In",
+      "SignOut": "Log Out"
+    },
+    "UserActivities": {
+      "ActivityName": "Form",
+      "Summary": "Summary",
+      "ActivityStatus": "Status",
+      "ActivityDate": "Created",
+      "ActivityActions": "Actions"
+    },
+    "ManageParticipants": {
+      "GovernedParticipants": "Governed participants",
+      "ParticipantPlaceholder": "Participant",
+      "ParticipantsList": "Participants"
+    },
+    "ParticipantProfile": {
+      "OpenProfile": "Profile",
+      "ViewAs": "View As",
+      "ManageParticipants": "Manage participants"
+    },
+    "UserPreferences": {
+      "UserPreferencesTitle": "User preferences",
+      "UserPreferencesGender": "Sex",
+      "UserPreferencesLocale": "Preferred language"
+    },
+    "Dashboard": {
+      "Title": "My Console",
+      "Text": ""
+    },
+    "DatePicker": {
+      "Day": "Choose day...",
+      "Month": "Choose month...",
+      "Year": "Choose year...",
+      "January": "January",
+      "February": "February",
+      "March": "March",
+      "April": "April",
+      "May": "May",
+      "June": "June",
+      "July": "July",
+      "August": "August",
+      "September": "September",
+      "October": "October",
+      "November": "November",
+      "December": "December"
+    },
+    "UserMenu": {
+      "DashboardButton": "Dashboard",
+      "SignOutButton": "Sign Out"
+    },
+    "DetailsPlaceholder": {
+      "PluralForm": "characters remaining",
+      "SingularForm": "character remaining"
+    },
+    "Title": {
+      "Default": "Osteosarcoma Project",
+      "Alert": "Your session will time-out soon"
+    },
```

Comment:
Osteosarcoma?
Please, review the translation files, I saw other mentions about Osteosarcoma. I think, there are a bunch of unused translations.

Reply:
fixed